general/1_terminal/Danylo_Kulbachinskiy/script4.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+
+
+jq '.messages[]' result.json | jq '.text' | tr -d 'a-z',"A-Z",'/"[]{}:\.','-_','&#'
```

Comment:
I like that you thought about making it prettier, but it mangled the output somewhat:
```
у кого есть конфиги вима  тоже скидывайте

  Я




  'ом пользоваться боюсь Там чтото страшное Плюс пока что меня спасал  +
 
 ```

I think it ate some of our text. The solution without `tr` would be accepted, but if you would like to try to make it prettier, you can do it, optionally.